.github/workflows/runpipe.yml

```diff
@@ -0,0 +1,30 @@
+name: Run runpipe tests
+on:
+  push:
+    branches-ignore:
+      - main
```

Comment:
Why do we ignore this on main and other branches?

Reply:
We have the merge train, so we already had a PR where we ran this.